R/RLearner_regr_randomForest.R

```diff
@@ -66,6 +68,18 @@ trainLearner.regr.randomForest = function(.learner, .task, .subset, .weights = N
   } else {
     data = getTaskData(.task, .subset, target.extra = TRUE)
     m = randomForest::randomForest(x = data[["data"]], y = data[["target"]], ...)
+    ## ugly hack
+    ## if the formula interface is used the training data is saved, but not with the default interface
+    ## the training data is used when predict.type = "se" and calibration is not FALSE
+    if (.learner$predict.type == "se") {
+      if (is.null(.learner$par.vals$calibrate)) {
+        m$data = data[["data"]]
+      } else {
+        if (.learner$par.vals$calibrate) {
+          m$data = data[["data"]]
```

Comment:
that is from a calibration procedure used in the infinitesimal jackknife which ensures that all variance estimates are positive, among other things. this requires the training data. `randomForestCI` never went to cran though, so are we including it now? i don't think the author has any intention of doing so. i could however put his code in mlr directly.